test/perf/benchmarks/Engine.ScriptBlock.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+using System.Runtime.InteropServices;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace Engine
+{
+    [BenchmarkCategory(Categories.Engine, Categories.Public)]
+    public class Scripting
+    {
+        private Runspace runspace;
+        private ScriptBlock scriptBlock;
+
+        private void SetupRunspace()
+        {
+            runspace = RunspaceFactory.CreateRunspace(InitialSessionState.CreateDefault2());
```

Comment:
Do we need `InitialSessionState.CreateDefault2()`? The method `CreateRunspace()` should also work. If `CreateDefault2` is needed, add a comment.

Reply:
Unless you are going to run some commands from other powershell modules, `CreateDefault2` is enough. I will add a comment to call it out.